### PR TITLE
Keep focus on the Composer when send button is clicked

### PR DESCRIPTION
**Pull request by parasharrajat**
### Details

https://github.com/Expensify/App/issues/7798#issuecomment-1062973852
### Fixed Issues

$ https://github.com/Expensify/App/issues/7798

#### Contributor (PR Author) Checklist
- [x] I verified the PR has a small number of commits behind `main`
- [x] I linked the correct issue in the `### Fixed Issues` section above
- [x] I wrote clear testing steps that cover the changes made in this PR
    - [x] I clearly indicated the environment tests should be run in (Staging vs Production)
- [x] I wrote testing steps that cover success & fail scenarios (if applicable)
- [x] I included screenshots or videos for tests on [all platforms](https://github.com/Expensify/App/blob/main/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [x] I ran the tests & veryfy they passed on:
    - [x] iOS / native
    - [x] Android / native
    - [x] iOS / Safari
    - [x] Android / Chrome
    - [x] MacOS / Chrome
    - [x] MacOS / Desktop
- [x] I verified there are no console errors related to changes in this PR
- [x] I followed proper code patterns (see [Reviewing the code](https://github.com/Expensify/App/blob/main/PR_REVIEW_GUIDELINES.md#reviewing-the-code))
    - [x] I added comments when the code was not self explanatory
    - [x] I put all copy / text shown in the product in all `src/languages/*` files (if applicable)
    - [x] I followed proper naming convention for platform-specific files (if applicable)
    - [x] I followed style guidelines (in [`Styling.md`](https://github.com/Expensify/App/blob/main/STYLING.md)) for all style edits I made
    - [x] I followed the JSDocs style guidelines (in [`STYLE.md`](https://github.com/Expensify/App/blob/main/STYLE.md#jsdocs))
- [x] I followed the guidelines as stated in the [Review Guidelines](https://github.com/Expensify/App/blob/main/PR_REVIEW_GUIDELINES.md)
- [x] I tested other components that can be impacted by my changes (i.e. if the PR modifies a shared library or component like `Avatar`, I verified the components using `Avatar` are working as expected)
- [x] I corroborated the UI performance was not affected (the performance is the same than `main` branch)
- [x] If I created a new component I verified that a similar component doesn't exist in the codebase

#### PR Reviewer Checklist
- [ ] I verified the PR has a small number of commits behind `main`
- [ ] I verified the correct issue is linked in the `### Fixed Issues` section above
- [ ] I verified testing steps are clear and they cover the changes made in this PR
    - [ ] I verified the testing environment is mentioned in the test steps
- [ ] I verified testing steps cover success & fail scenarios (if applicable)
- [ ] I checked that screenshots or videos are included for tests on [all platforms](https://github.com/Expensify/App/blob/main/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [ ] I verified tests pass on **all platforms** & I tested again on:
    - [ ] iOS / native
    - [ ] Android / native
    - [ ] iOS / Safari
    - [ ] Android / Chrome
    - [ ] MacOS / Chrome
    - [ ] MacOS / Desktop
- [ ] I verified there are no console errors related to changes in this PR
- [ ] I verified proper code patterns were followed (see [Reviewing the code](https://github.com/Expensify/App/blob/main/PR_REVIEW_GUIDELINES.md#reviewing-the-code))
    - [ ] I verified comments were added when the code was not self explanatory
    - [ ] I verified any copy / text shown in the product was added in all `src/languages/*` files (if applicable)
    - [ ] I verified proper naming convention for platform-specific files was followed (if applicable)
    - [ ] I verified [style guidelines](https://github.com/Expensify/App/blob/main/STYLING.md) were followed
    - [ ] I verified the JSDocs style guidelines (in [`STYLE.md`](https://github.com/Expensify/App/blob/main/STYLE.md#jsdocs)) were followed
- [ ] I verified that this PR follows the guidelines as stated in the [Review Guidelines](https://github.com/Expensify/App/blob/main/PR_REVIEW_GUIDELINES.md)
- [ ] I verified other components are not impacted by changes in this PR (i.e. if the PR modifies a shared library or component like `Avatar`, I verified the components using `Avatar` are working as expected)
- [ ] I verified the UI performance was not affected (the performance is the same than `main` branch)
- [ ] If a new component is created I verified that a similar component doesn't exist in the codebase

### QA Steps
1. Open any chat on each platforms.
2. Type a message in composer. 
3. manually click the send button. 
4. Notice that cursor should remain on the Composer after message is sent. 
5. On Mobile devices(native or mWeb) Virtual keyboard should not close after message is sent.

- [ ] Verify that no errors appear in the JS console

#### Web | Desktop

https://user-images.githubusercontent.com/24370807/157866148-3769724d-0d91-4fca-935a-af6d99cb5fd8.mp4

#### Mobile Web

https://user-images.githubusercontent.com/24370807/157866185-aab45ac5-f7b4-4f86-9157-1a9022de8e6a.mp4

https://user-images.githubusercontent.com/24370807/157883113-94f502c6-b140-414c-a9d8-cad23520fd17.mp4

#### iOS

https://user-images.githubusercontent.com/24370807/157881126-1a24705c-b624-4a14-bd2d-844356d4c9b5.mp4

#### Android

https://user-images.githubusercontent.com/24370807/157866172-56b96bfb-5d6d-4b96-ba3a-bb52b28dcbb5.mp4

### CODE REVIEW

File: src/pages/home/report/ReportActionCompose.js [hunk 1]
```diff
@@ -578,6 +581,10 @@ class ReportActionCompose extends React.Component {
                                 ]}
                                 onPress={this.submitForm}
                                 underlayColor={themeColors.componentBG}
+
+                                // Keep focus on the composer when Send message is clicked.
+                                // eslint-disable-next-line react/jsx-props-no-multi-spaces
```
Comment (roryabraham) [inline src/pages/home/report/ReportActionCompose.js:586]: We should disable this eslint rule

**Review by roryabraham — APPROVED** [review]

File: src/pages/home/report/ReportActionCompose.js (hunk 1 above)
Comment (parasharrajat) [reply src/pages/home/report/ReportActionCompose.js:586]: Yeah, I guess so. It is used in many places

**Comment by parasharrajat** [comment]
Yup, I think this PR is behind that.

**Comment by parasharrajat** [comment]
> So I guess the next step is to revert this PR and fix the original issue with a different approach?

No. is the new issue a deploy blocker? 

The regression period for this issue is over. so it would be best to fix the new issue separately.

**Pull request merged**